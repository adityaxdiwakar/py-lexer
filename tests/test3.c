#include "stdio.h"

int main() {
    int x;
    int y;
    int max;
    scanf("%d %d",&x,&y);
    if (x > y) {
        max = x;
    } else {
        max = y;
    }
    printf("%d",max);
}
