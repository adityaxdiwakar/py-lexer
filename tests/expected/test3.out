DIRECTIVE
STR
INT_WORD
IDENT(main)
LBRACK
INT_WORD
IDENT(x)
SEMI
INT_WORD
IDENT(y)
SEMI
INT_WORD
IDENT(max)
SEMI
IDENT(scanf)
LPAREN
STR
COMMA
IDENT(x)
COMMA
IDENT(y)
RPAREN
SEMI
IF
LPAREN
IDENT(x)
GT
IDENT(y)
RBRACK
IDENT(max)
EQUALS
IDENT(x)
SEMI
RBRACK
ELSE
LBRACK
IDENT(max)
EQUALS
IDENT(y)
SEMI
RBRACK
IDENT(printf)
STR
COMMA
IDENT(max)
SEMI
RBRACK